rts/Sim/Objects/SolidObject.cpp

```diff
@@ -448,8 +449,8 @@ void CSolidObject::Kill(CUnit* killer, const float3& impulse, bool crushed)
 
 
 
-float CSolidObject::CalcFootPrintMinExteriorRadius(float scale) const { return ((math::sqrt((xsize * xsize + zsize * zsize)) * 0.5f * SQUARE_SIZE) * scale); }
-float CSolidObject::CalcFootPrintMaxInteriorRadius(float scale) const { return ((std::max(xsize, zsize) * 0.5f * SQUARE_SIZE) * scale); }
+float CSolidObject::CalcFootPrintMinExteriorRadius(float scale) const { return ((pathRadiusMult * math::sqrt((xsize * xsize + zsize * zsize)) * 0.5f * SQUARE_SIZE) * scale); }
```

Comment:
Since you are just multiplying the `math::sqrt((xsize * xsize + zsize * zsize))` by a factor, why is this different from just simply increasing the footprint?